pom.xml

```diff
@@ -17,8 +17,7 @@
         <dependency>
             <groupId>net.dv8tion</groupId>
             <artifactId>JDA</artifactId>
-            <version>3.0.0_178</version>
-            <scope>compile</scope>
+            <version>3.1.0_206</version>
```

Comment:
A) [I have already fixed this.](https://github.com/JDA-Applications/JDA-Utilities/pull/14/commits/c862cf1e7c883b7fb7ec002869f05a26cb88e8f7)
B) I know nothing about Maven, and the only reason I fixed it the way I did was because my IDE automatically deleted `<scope>compile</scope>` before I made this commit.
@jagrosh will have to change it himself, otherwise I'll leave it be.